.github/workflows/tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       max-parallel: 5
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9, '3.10']
+        python-version: [3.7, 3.8, 3.9, '3.10']
```

Comment:
3.7 is EoL quite soon, in 7 months I think? Since this change is backwards-incompatible, should we also drop 3.7 already?

Reply:
It’s more like 9 months till the end of June 2023. So I think it would be premature.